### DISCUSSION TITLE
v0.9 released!

### DISCUSSION THREAD

**Discussion by baffler** [baffler/Transparent-Twitch-Chat-Overlay#63] [General]
https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.9

New feature for this version, you can now filter chat by usernames, allow all mods, or allow all VIPs (or any of that combination). Should be really useful for very active chats.

**Comment by carnifexx** [comment]
great work!
(its still says 0.81 in the "about" section of the settings.)

**Comment by HachirokuGG** [comment]
I noticed that on some games like apex or when i play other games that it doesn't totally in game when it starts it also applies to when i play other games besides apex that the overlay is no where to be found only when i tab out

**Comment by AlastorAlex** [comment]
Thank you SO MUCH for this application, now i dont have to play everything on windowed 720p! works wonderfull!
